game/builders/CharacterBuilder.cpp

```diff
@@ -24,9 +25,25 @@ namespace builders {
         // Create entity and define position and dimension
         auto& player = m_ecsWorld.createEntity();
 
-        // TODO: Use the character spawn point position
-        common::Vector2D<double> position{ 5, 13 };
+        // Set default position for player (when no character spawns are present)
         common::Vector2D<double> dimension{ 1, 2 };
+        common::Vector2D<double> position{ m_physicsWorld.getSize().castTo<double>() };
```

Comment:
You should be able to use 
```suggestion
        common::Vector2D<double> position{ m_physicsWorld.getSize().castTo<double>() / 2 };
```
here to avoid two extra lines :)